app/scripts/controllers/eventDetails.js

```diff
@@ -86,6 +87,16 @@ angular
     $scope.originalConference = conference;
     $scope.conference = angular.copy(conference);
     $scope.currencies = currencies;
+    $scope.countries = allCountries;
+    $scope.conference.locationCountry = conference.locationCountry
+      ? conference.locationCountry
+      : 'US';
+    $scope.currentRegions =
+      $scope.countries[
+        $scope.countries
+          .map(c => c[1])
+          .indexOf($scope.conference.locationCountry)
+      ][2];
```

Comment:
Dealing with indexes seems a little odd here. Idk if this code is perfect but seems like you could just find the country directly:
```suggestion
    $scope.currentRegions =
      $scope.countries.find(c => c[1] === $scope.conference.locationCountry)?.[2];
```

Reply:
Yeah I agree, this is a better approach.